addons/redis/templates/componentdefinition-redis-sentinel.yaml

```diff
@@ -76,6 +76,23 @@ spec:
           name: default
           optional: false
           password: Required
+    ## the default password of redis connection
+    - name: REDIS_DEFAULT_PASSWORD
+      valueFrom:
+        credentialVarRef:
+          ## reference the current component definition name
+          compDef: {{ include "redis.componentDefName" . }}
+          name: default
+          optional: true
+          password: Required
+          multipleClusterObjectOption:
```

Comment:
will each pod have different  password&secrets?

Reply:
> will each pod have different password&secrets?

This is not for each pod, but for each shard. For multiple groups of Redis primary-replica shards, they can have the same password or different passwords, depending on the business settings. This password is used by Sentinel to connect to the Redis server.